test/test_smoke.py

```diff
@@ -95,22 +100,48 @@ def image_type_fixture(tmpdir_factory, build_container, request):
     config_json_path.write_text(json.dumps(CFG), encoding="utf-8")
 
     cursor = testutil.journal_cursor()
-    # run container to deploy an image into output/qcow2/disk.qcow2
-    subprocess.check_call([
-        "podman", "run", "--rm",
-        "--privileged",
-        "--security-opt", "label=type:unconfined_t",
-        "-v", f"{output_path}:/output",
-        "-v", "/store",  # share the cache between builds
-        build_container,
-        "quay.io/centos-bootc/fedora-bootc:eln",
-        "--config", "/output/config.json",
-        "--type", image_type,
-    ])
+
+    upload_args = []
+    creds_args = []
+    with tempfile.TemporaryDirectory() as tempdir:
+        if image_type == "ami":
```

Comment:
we probably need a check there that we actually have AWS credentials, when running this on a system without we should skip this part of the test (but not the ami generation as this is useful with qemu still)

Reply:
Not sure about this.  Currently it will fail, which can be annoying if you want to test the AMI in a scenario where you don't have an AWS account but intend to boot it with QEMU.  But if we skip the test instead, then a config mixup in CI will hide the issue by ending up green.